caravel/assets/javascripts/SqlLab/main.css

```diff
@@ -233,6 +241,10 @@ div.tablePopover:hover {
     padding-top: 10px;
 }
 
+.TableElement {
```

Comment:
it looks like the scroll bar is still overlapping some text in the panel, can we give it some space between the content and the scrollbar so it isn't covering any content?